sklearn/utils/estimator_checks.py

```diff
@@ -396,6 +399,9 @@ def set_checking_parameters(estimator):
     if name == 'OneHotEncoder':
         estimator.set_params(handle_unknown='ignore')
 
+    if name == 'VotingClassifier':
+        estimator.set_params(voting='soft')
```

Comment:
Please add a comment explaining that we need this for `predict_proba` to be defined